src/stream/src/task/barrier_manager/managed_state.rs

```diff
@@ -134,7 +145,7 @@ impl ManagedBarrierState {
 
     /// Remove stop barrier (epoch < `curr_epoch`), and send err.
     pub(crate) fn remove_stop_barrier(&mut self, curr_epoch: u64) {
-        self.epoch_barrier_state_map.retain(|k, _| k > &curr_epoch);
+        self.epoch_barrier_state_map.retain(|k, _| k >= &curr_epoch);
```

Comment:
yes, because we are now sorted to proceed `sealed_epoch` in `managed_state`. So we make sure that all the ones less than or equal to`barrier.prev_epoch` are cleared when we do `stop_all_actors`. If not, We will be blocked in `collect_barrier`

Reply:
we should use `>`, And I use `drain_collect_rx` before `send_barrier` in `stop_all_actors` to avoid blocking `collect_barrier` calls